Directory.Build.targets

```diff
@@ -1,5 +1,17 @@
 <Project>
   <!-- Properties which should be set after the project has been evaluated -->
+
+  <PropertyGroup>
+    <!--
+      By default, all projects which produce packages are not intended to ship to NuGet.org as a product package.
+      Packages which are intended to ship to NuGet.org must opt-in by setting this to true in the project file.
+    -->
+    <IsProductPackage Condition=" '$(IsProductPackage)' == '' ">false</IsProductPackage>
```

Comment:
Is this semantically different from `IsShipping` that's used by arcade? If there's an important different should we discuss with them? If not, should we just use the same property name as arcade?

Reply:
I think of IsShipping as a subset of IsProductPackage. IsProductPackage means "this is a package that is meant for a customer to use" and IsShipping means "this package should ship in the next release". In 2.2 we had a handful of packages that were built as "product packages" but that we decided to yank from shipping after the build was done.

Reply:
It sounds like we have a requirement that they don't totally support then? Is this tracked somewhere

Reply:
https://github.com/dotnet/arcade/issues/1179 is probably the best item for this. Arcade publishing is a hot mess right now and will likely be completely re-invented again before we finish RTM. I'm not pinning any hopes on 'IsShipping'.